oaknational/Oak-Web-Application
# Adding cycle 2 to curriculum visualiser behind flag

## Description
Added cycle 2 data into the visualiser behind a flag

## Issue(s)
`CUR-800`

## How to test

1. Go to https://deploy-preview-2692--oak-web-application.netlify.thenational.academy/teachers/curriculum
2. Check for regressions

Enable cycle 2 locally and test we're pulling though the data. Probably some issues at this point, we can raise further tickets as long as the data is essentially there. 

## Screenshots
<img width="375" alt="Screenshot 2024-08-13 at 17 20 04" src="https://github.com/user-attachments/assets/8230749f-ff65-43f0-a577-5064af572144"> <img width="395" alt="Screenshot 2024-08-13 at 17 19 42" src="https://github.com/user-attachments/assets/8a14a63f-29a9-4fac-a387-029576ae2fbd">
